src/components/PopupModal.js

```diff
@@ -4,7 +4,6 @@ import {ReactComponent as SearchIcon} from '../icons/search.svg';
 import {ReactComponent as ClearIcon} from '../icons/clear.svg';
 import {ReactComponent as CircleAnimated} from '../icons/circle-anim.svg';
 import {useContext, useEffect, useMemo, useRef, useState} from 'react';
-import {getBundledCssLink} from '../utils/helpers';
```

Comment:
Is the expectation that people working on sodo-search need to set the URLs in their config for CSS? I guess we already require that for JS, right?

Reply:
> Is the expectation that people working on sodo-search need to set the URLs in their config for CSS

It wasn't yet but will be now yeah, if people want to use the their local sodo-search CSS they'd need to set it in the config. I was thinking about it that we could also manage to create CSS Url if missing from the JS `url`, but it might lead to more confusion down the line so its better to expect them to add the CSS url like JS once, wdyt ?

Reply:
SGTM 👍🏻